packages/core/src/app/common/error/SentryErrorLogger.spec.ts

```diff
@@ -154,45 +161,13 @@ describe('SentryErrorLogger', () => {
 
         expect(init)
             .toHaveBeenCalledWith(expect.objectContaining({
-                blacklistUrls: [
+                denyUrls: [
                     'polyfill~checkout',
                     'sentry~checkout',
                 ],
             }));
     });
 
-    it('does not rewrite filename of error frames if it does match with public path', () => {
-        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
-        new SentryErrorLogger(config, { publicPath: 'https://cdn.foo.bar' });
-
-        const clientOptions: BrowserOptions = (init as jest.Mock).mock.calls[0][0];
-
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        const rewriteFrames = (clientOptions.integrations! as Integration[]).find(integration => (
-            integration.name === 'RewriteFrames'
-        )) as RewriteFrames;
-
-        const output = rewriteFrames.process({
-            stacktrace: {
-                frames: [
-                    {
-                        colno: 1234,
-                        filename: 'https://cdn.hello.world/js/app-123.js',
-                        function: 't.<anonymous>',
-                        in_app: true,
-                        lineno: 1,
-                    },
-                ],
-            },
-        });
-
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        const frame = output.stacktrace!.frames![0];
-
-        expect(frame)
-            .toEqual(frame);
```

Comment:
Based on Sentry source code, they will rewrite Filename even if it does match with public path.
https://github.com/getsentry/sentry-javascript/blob/487810a32d6d6c5356386ea0b55b84a85675cb04/packages/integrations/src/rewriteframes.ts#L70